robyn/__main__.py

```diff
@@ -6,7 +6,16 @@
 
 def create():
     project_dir = input("Enter the name of the project directory: ")
+    docker = input("Need Docker? (Y/N) ")
+
     # Initailize a new Robyn project
+    def check(value):
+        while value not in ["Y", "N"]:
```

Comment:
@Oluwaseun241 , I don't think I want to a loop here. Can you add an `if` check to see if the value is not equal to `Y` or `y`, we replace it with `N`

Reply:
The loop was there because I don't want them to restart the CLI when there is a mistakenly or genuinely unknown command